scene/gui/rich_text_label.cpp

```diff
@@ -2907,6 +2964,19 @@ Dictionary RichTextLabel::parse_expressions_for_values(Vector<String> p_expressi
 	return d;
 }
 
+Color RichTextLabel::parse_color(const String &p_color) {
+	Color color;
+
+	if (Color::html_is_valid(col))
```

Comment:
The variable `col` is not declared. Instead, there should be a `p_color`. :smile:

Reply:
Thank you for checking :smiley: